Yoast/javascript
# Remove global css rules

## Summary


This PR can be summarized in the following changelog entry:

* Removes some global css rules

## Relevant technical choices:

* We removed the global styling (e.g. "yoast p") to prevent conflicts on parts of the plugin that do not have yet the new styling.

## Test instructions

This PR can be tested by following these steps:

* Make sure to have the monorepo linked with the plugin
* Run `grunt build` in the plugin
* Check in the plugin if the select2 components have the new styling
* Check in the plugin that all other components like input fields and radio buttons have the old styling
* Check on the monorepo if the select.html demo works as expected to the new styling

## Impact check

* This PR affects the following parts of the plugin, which may require extra testing:
  *

## UI changes

## Quality assurance


Fixes #
